### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ivanzh13**

### CODE REVIEW

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/cafe.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        vaccine = visitor.get("vaccine")
+        if not vaccine:
+            raise NotVaccinatedError("Visitor does't have vaccine.")
+        if date.today() > vaccine["expiration_date"]:
```
Comment (vasylhnatiuk) [inline app/cafe.py:17]: use get() here also

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

File: app/cafe.py (hunk 1 above)
Comment (ivanzh13) [reply app/cafe.py:17]: If I get None, I still get an error

Comment (ivanzh13) [reply app/cafe.py:17]: Do you can explain me, why use get here

Comment (vasylhnatiuk) [reply app/cafe.py]: Here is more about style of code: if you use get() in one place, use it in all places. And you will not have KeyError

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**